cord/client.py

```diff
@@ -827,10 +824,7 @@ def get_data(self, data_hash: str, get_signed_url: bool = False) -> Tuple[Option
 
         images: Union[List[Image], None] = None
         if dataset_data['images'] is not None:
-            images = []
-
-            for image in dataset_data['images']:
-                images.append(Image(image))
+            images = [Image(image) for image in dataset_data['images']]
```

Comment:
Function `CordClientProject.get_data` refactored with the following changes:

- Convert for loop into list comprehension ([list-comprehension](https://docs.sourcery.ai/refactorings/list-comprehension/))